src/components/v2v3/V2V3Project/V2V3FundingCycleSection/modals/DistributePayoutsModal.tsx

```diff
@@ -150,45 +131,12 @@ export default function DistributePayoutsModal({
       width={640}
     >
       <Space direction="vertical" size="large" style={{ width: '100%' }}>
-        <div>
-          <div style={{ display: 'flex', justifyContent: 'space-between' }}>
-            <Trans>
-              Total funds:{' '}
-              <div>
-                <CurrencySymbol currency={distributionCurrencyName} />
-                {grossAvailableAmount}
-              </div>
-            </Trans>
-          </div>
-
-          <div style={{ display: 'flex', justifyContent: 'space-between' }}>
-            <div>
-              <Trans>JBX Fee ({feePercentage}%):</Trans>
-            </div>
-            <div>
-              - <CurrencySymbol currency={distributionCurrencyName} />
-              {feeAmount}
-            </div>
-          </div>
-
-          <Divider style={{ margin: '4px 0' }} />
-
-          <div
-            style={{
-              display: 'flex',
-              justifyContent: 'space-between',
-              fontWeight: 500,
-            }}
-          >
-            <div>
-              <Trans>Available after fee:</Trans>
-            </div>
-            <div>
-              <CurrencySymbol currency={distributionCurrencyName} />
-              {netAvailableAmountFormatted}
-            </div>
-          </div>
-        </div>
+        <p>
+          <Trans>
+            Distribute your project's funds. Distribution to addresses other
+            than other Juicebox projects will incur a 2.5% platform fee.
```

Comment:
We use this text elsewhere. might be good to be consistent.

```suggestion
            Distributions to Ethereum addresses incur a 2.5% JBX membership fee.
```